docs/developer-guide/udf.rst

```diff
@@ -357,8 +364,8 @@ used to call the UDAF. The UDAF can be invoked in four ways:
 
       @UdafFactory(description = "sums longs")
       // Can be used with table aggregations
-      public static TableUdaf<Long, Long> createSumLong() {
-        return new TableUdaf<Long, Long>() {
+      public static TableUdaf<Long, Long, Long> createSumLong() {
```

Comment:
Consider switching the example UDF to one that has a different intermediate type, e.g.  `AVG`, as this will demo the new functionality and call out the use of `aggregateSchema`.